Docs/source/usage/workflows.rst

```diff
@@ -8,11 +8,12 @@ This section collects typical user workflows and best practices for WarpX.
 .. toctree::
    :maxdepth: 2
 
+   workflows/extend
```

Comment:
To be more precise, perhaps the file should be called something like "extend_with_python.rst". Who knows, perhaps in the future someone might implement "extend_with_julia.rst".